src/main/java/fr/openent/viescolaire/security/AccessStructureMyCourse.java

```diff
@@ -0,0 +1,42 @@
+package fr.openent.viescolaire.security;
+
+import fr.openent.viescolaire.service.ClasseService;
+import fr.openent.viescolaire.service.CommonCoursService;
+import fr.openent.viescolaire.service.impl.DefaultClasseService;
+import fr.openent.viescolaire.service.impl.DefaultCommonCoursService;
+import fr.wseduc.webutils.http.Binding;
+import fr.wseduc.webutils.http.Renders;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.entcore.common.http.filter.ResourcesProvider;
+import org.entcore.common.user.UserInfos;
+
+public class AccessStructureMyCourse implements ResourcesProvider {
+    public static final Logger log = LoggerFactory.getLogger(Renders.class);
```

Comment:
Attention au nom du log de la classe

```suggestion
    public static final Logger log = LoggerFactory.getLogger(AccessStructureMyCourse .class);
```

Reply:
Fait